TWLight/templates/base.html

```diff
@@ -254,6 +254,12 @@
               {% trans "Feedback" %}
             </a>
           </li>
+          <li>
+            <a href="https://translatewiki.net/wiki/Translating:Wikipedia_Library_Card_Platform">
+              {% comment %} Translators: This button is at the bottom of every page and can be clicked by users to navigate to TranslateWiki Wikipedia Library Card platform translation page. {% endcomment %}
+              {% trans "Translate" %}
```

Comment:
I'd recommend changing this into the unlocalized project name, e.g. `translatewiki.net`
```suggestion
```